src/sass/partials/_globals.scss

```diff
@@ -25,10 +25,14 @@ a {
   }
 }
 .button {
-  min-height: 40px;
+  max-height: 44px;
   text-align: center;
-  border-radius: 20px;
+  border-radius: 25px;
   border: none;
+  padding: 0 37px;
+  @include phone-max {
```

Comment:
to jednak uzywamy phone?

Reply:
Ja bym zostawił w mixinach, bo czasem   się przydaje do niektórych styli, szczególnie, że wersja do 480px ma być responsywna. ja też dodałem kilka mixinów na swój użytek i dopóki się ich nie wywoła w specyficznych miejscach to nic nie zepsują